scopes/dependencies/modules/packages-excluder/generate-node-modules-pattern.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { generateNodeModulesPattern } from './generate-node-modules-pattern';
+
+describe('generateNodeModulesPattern()', () => {
```

Comment:
you should check somewhere that for other packages you indeed get true, at least once. so for example `expect(regex.test('node_modules/not-excluded-package/some-path')).toBeTruthy();` in this case not-excluded-package with some-path should return true